src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<{ movies: Movie[] }> = ({ movies }) => (
```

Comment:
Use interface for passing props, it will clean up your code. 
Also you can import `FC` directly from ''react"
```suggestion
interface Props {
  movies: Movie[]
}
export const MoviesList: FC<Props> = ({ movies }) => (
```